src/components/Header/Header.style.js

```diff
@@ -1,13 +1,17 @@
 import styled from 'styled-components';
 
 export const Wrapper = styled.header`
-  padding-left: 80px;
-  padding-right: 80px;
-  padding-top: 32px;
+  width: 100%;
+  height: 100px;
+  display: flex;
+  justify-content: space-between;
 `;
 
 export const Nav = styled.nav`
+  width: 100%;
   display: flex;
+  padding-left:80px;
+  padding-right: 54px;
```

Comment:
It took me a moment to understand why the left and right padding are different. This Is totally fine. Just as an alternative option you could use the same padding here and style the `Li` component with

```
export const Li = styled.li`
  font-size: 16px;
  margin-right: 26px;

  &:last-child {
    margin-right: 0;
  }
`;
```

No need to change your implementation though. Just a hint in case you didn't know 😉

Reply:
awesome ok, duly noted 💪 👍